.gitignore

```diff
@@ -23,4 +23,8 @@ coverage/
 /tmp
 
 conferences_data.rb
-conferences_raffle.txt
\ No newline at end of file
+conferences_raffle.txt
+
+Gemfile
```

Comment:
My Gemfile contains sqlite3 gem that is not configured originally by the app. That's why I put it in the ignore